drafts/use-cases/1.entry-point.md

```diff
@@ -0,0 +1,108 @@
+## Entry point
+
+### Story
+As an application user
+I want my application to consume an entry point for the Web API by providing a base URL
+So I can start managing my events.
+
+### Usage
+```javascript
+var client = new HydraClient();
+var data = client.get("/api");
```

Comment:
This should probably an absolute URL 

    client.get("http://www.example.com/api"); 

Even more interesting, or at least worth mentioning as well, would be 

    client.get("http://www.example.com/"); 

which then uses the Hydra API documentation (`hydra:entrypoint`) to find the API's `http://www.example.com/api`.